README.md

```
@@ -0,0 +1,358 @@
+   - [Slide - Pdf](https://github.com/chandankushwahaa/100xdevs_2.0/blob/main/0-1/week07/Week%207.2%20Context%20API%2C%20State%20Management%20%26%20Recoil.pdf)
+
+- Week 7.3 Recoil Deep Drive
+
+***[CODE](https://github.com/chandankushwahaa/100xdevs_2.0/tree/main/0-1/week07/mynotes)***
+
+<hr></hr>
+
+## Week - 8 Tailwind, Paytm Clone
+
+- Week 8.1 Tailwind
+   - [Slide - Pdf](https://github.com/chandankushwahaa/100xdevs_2.0/blob/main/0-1/week08/8%20Tailwind.pdf)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)